src/resources/offices.js

```diff
@@ -22,5 +22,9 @@ export const offices = createResourceDescription(
     subjectSettings: createResourceDescription('subject_settings', {
       filters: ['subject_id'],
     }),
+    availabilityHours: createResourceDescription('availability_hours'),
```

Comment:
When I tried this, it didn't work, I don't think this syntax is currently supported in the api.

Reply:
Can the availability hour exceptions also be removed?

Reply:
No, that one does work.